web/src/routes/(user)/map/+page.svelte

```diff
@@ -8,17 +8,25 @@
 		isViewingAssetStoreState,
 		viewingAssetStoreState
 	} from '$lib/stores/asset-interaction.store';
+	import { api, MapMarkerResponse } from '@api';
+	import { onMount } from 'svelte';
+	import { browser } from '$app/environment';
 
 	export let data: PageData;
+	let mapMarkers: MapMarkerResponse[] = [];
 
 	let initialMapCenter: [number, number] = [48, 11];
 
-	$: {
-		if (data.mapMarkers.length) {
-			let firstMarker = data.mapMarkers[0];
-			initialMapCenter = [firstMarker.lat, firstMarker.lon];
+	onMount(async () => {
+		mapMarkers = MapMarkerResponse.fromMany(data.mapMarkers);
+
+		if (mapMarkers.length) {
+			initialMapCenter = mapMarkers[0].getLatLon();
 		}
-	}
+
+		const markersResponse = (await api.assetApi.getMapMarkers()).data;
```

Comment:
Everywhere else in the code we use destructuring. Eg `const { data: markersResponse } = await ...`